ingestion/tests/unit/test_sql_lineage.py

```diff
@@ -0,0 +1,36 @@
+#  Copyright 2021 Collate
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#  http://www.apache.org/licenses/LICENSE-2.0
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+"""
+sql lineage utils tests
+"""
+
+from unittest import TestCase
+
+from sqllineage.runner import LineageRunner
+
+from metadata.utils.sql_lineage import populate_column_lineage_map
+
+QUERY = """create view test2 as
+SELECT brand_id
+FROM production.brands;"""
+
+EXPECTED_LINEAGE_MAP = {
```

Comment:
could we get a few extra examples here? It would be an interesting opportunity to list down all types of queries that we support (as well as the queries that we don't :) ) We can then link this in the docs or something to help users understand how this works

Reply:
Sure @pmbrull i will add.

Reply:
thanks mate